backend/app/services/knowledge/orchestrator.py

```
@@ -0,0 +1,900 @@
+            document_id=document.id,
+            kb_index_info=None,
+            trace_context=None,
+        )
+
+    def _schedule_summary(
+        self,
+        db: Session,
+        knowledge_base: Kind,
+        document: Any,
+        user: User,
+        async_callback: Callable,
```

Ruff (0.14.14)

[warning] Unused method argument: `async_callback`

(ARG002)